### PR TITLE
bugfix/23 - Ensure focus is on the command input

### LINKED ISSUE #23: Command bar cursor not at end of line in MS Edge
The cursor should be at the end of the line, but in Microsoft Edge, it is at the far left instead.

Originally reported in zlsa/atc#489.

Also, per duplicate issue #1438, clicking aircraft will sometimes not put the command bar in focus, meaning the first keystroke is not shown in the command bar.

### PR DESCRIPTION
**Pull request by oobayly**
Resolves #23 

The purpose of this pull request is to ensure that the command input textbox has focus after an aircraft is selected.

### CODE REVIEW

File: src/assets/scripts/client/InputController.js [hunk 1]
```diff
@@ -307,6 +307,9 @@ export default class InputController {
         this.input.callsign = aircraftModel.callsign;
         this.input.command = '';
         this.$commandInput.val(`${aircraftModel.callsign} `);
+        if (!this.$commandInput.is(':focus')) {
```
Comment (n8rzz) [inline src/assets/scripts/client/InputController.js]: There should be an empty line above this `if`, and also an empty line after the closing `}`

Comment (oobayly) [reply src/assets/scripts/client/InputController.js]: Indeed it should.

**Comment by oobayly** [comment]
I could also go with comparing it to document.activeElement, but it seems a bit clunky combining both vanilla js and jQuery.

**Comment by n8rzz** [comment]
> I could also go with comparing it to document.activeElement, but it seems a bit clunky combining both vanilla js and jQuery.

nah, we already have jquery here we may as well keep to the native solutions

**Pull request merged**